apps/genrsa.c

```diff
@@ -30,12 +30,14 @@ NON_EMPTY_TRANSLATION_UNIT
 # define DEFBITS 2048
 # define DEFPRIMES 2
 
+static int verbose = 0;
```

Comment:
I know that things look different in other programs...  in *this* program, you may notice that all option variables are defined in `genrsa_main`, so it would be nice if this was as well.

(we aren't consistent in this regard, I know...  that's a cleanup job waiting to happen.  It's not high priority, though)

Reply:
> I know that things look different in other programs... in this program, you may notice that all option variables are defined in genrsa_main, so it would be nice if this was as well.

`verbose` is accessed from both `genrsa_main()` and from `genrsa_cb()`.